README.md

```diff
@@ -1,8 +1,8 @@
 # VM Console Proxy
 
-Proxy that provides access to the VNC console of a Kubevirt VM.
+The implementation of this project has changed, and now it's not a proxy anymore.
 
-It can generate time limited tokens that are then used to access VNC.
+It provides an endpoint to generate time limited tokens that are then used to access VNC.
```

Comment:
I'd write it as:
```
# VM Console Proxy

Provides an endpoint to generate time limited tokens that are then used to access VNC.

**Note:*** This project no longer provides VNC proxy functionality, so the project name should be changed in the future.
```

Reply:
I wrote it differently. If you want, I can change it.